scanner/cmd/scanner/main.go

```diff
@@ -103,19 +112,34 @@ func createGRPCService(backends *Backends) (grpc.API, error) {
 	if err != nil {
 		return nil, fmt.Errorf("identity extractor: %w", err)
 	}
-	// Create gRPC API service.
+	// Create gRPC API service and debug routes.
+	var customRoutes []routes.CustomRoute
+	for path, handler := range routes.DebugRoutes {
+		customRoutes = append(customRoutes, routes.CustomRoute{
+			Route:         path,
+			Authorizer:    allow.Anonymous(),
```

Comment:
Central only allows admins to view this: https://github.com/stackrox/stackrox/blob/4.1.1/central/main.go#L827

This is interesting because Scanner doesn't really have this concept of users with roles. I think it might be open to all in Sensor. I'm not completely sure, though

Reply:
Anonymous is all right; We don't need fine control over each route, and we can control access with policies as needed.

Reply:
we do have the network policy, so LGTM